test/cpp/jit/test_gpu.cpp

```diff
@@ -2626,7 +2626,6 @@ TEST(NVFuserTest, FusionUnaryOps_CUDA) {
       OpTuple{at::expm1, UnaryOpType::Expm1, "expm1"},
       OpTuple{at::floor, UnaryOpType::Floor, "floor"},
       OpTuple{at::frac, UnaryOpType::Frac, "frac"},
-      // OpTuple{at::gelu, UnaryOpType::Gelu, "gelu"},
```

Comment:
Note. we would not want to break onnx export.
Would is be safe to convert it during onnx export and spit out vanilla gelu without the approximation?

Reply:
NVM... sorry I was looking at the wrong code.